bin/generate-podspecs.sh

```diff
@@ -128,82 +137,11 @@ do
-        echo "   ==> Patching $pod podspec"
-        # First move it to its own folder
-        mkdir -p "$DEST/FBReactNativeSpec"
-        mv "$DEST/FBReactNativeSpec.podspec.json" "$DEST/FBReactNativeSpec"
-
-        # Then we generate FBReactNativeSpec-generated.mm and FBReactNativeSpec.h files.
-        # They are normally generated during compile time using a Script Phase in FBReactNativeSpec added via the `use_react_native_codegen` function.
-        # This script is inside node_modules/react-native/scripts folder. Since we don't have the node_modules when compiling WPiOS,
-        # we're calling the script here manually to generate these files ahead of time.
-        SCHEMA_FILE="$TMP_DEST/schema.json"
-        NODE_BINARY="${NODE_BINARY:-$(command -v node || true)}"
-
-        if [ -d "$CODEGEN_REPO_PATH" ]; then
-            CODEGEN_PATH=$(cd "$CODEGEN_REPO_PATH" && pwd)
-        elif [ -d "$CODEGEN_NPM_PATH" ]; then
-            CODEGEN_PATH=$(cd "$CODEGEN_NPM_PATH" && pwd)
-        else
-            echo "Error: Could not determine react-native-codegen location. Try running 'yarn install' or 'npm install' in your project root." 1>&2
-            exit 1
-        fi
-
-        if [ ! -d "$CODEGEN_PATH/lib" ]; then
-            describe "Building react-native-codegen package"
-            bash "$CODEGEN_PATH/scripts/oss/build.sh"
-        fi
-
-        # Generate React-Codegen
-        # A copy of codegen_utils.rb is done to modify the content within get_react_codegen_spec
-        # this enables getting the schema for React-Codegen in runtime by printing the content.
-        echo "Generating React-Codegen"
-        REACT_NATIVE_CODEGEN_UTILS_PATH="$SCRIPTS_PATH/cocoapods/codegen_utils.rb"
-        REACT_NATIVE_CODEGEN_UTILS_MODIFIED_PATH="$SCRIPTS_PATH/cocoapods/codegen_utils_modified.rb"
-        # Making a temp copy of codegen_utils.rb
-        cp $REACT_NATIVE_CODEGEN_UTILS_PATH $REACT_NATIVE_CODEGEN_UTILS_MODIFIED_PATH
-        # Manually add the min_ios_version_supported variable to the CodegenUtils class
-        # The modified script won't be able to detect min_ios_version_supported otherwise
-        echo "
-        def min_ios_version_supported
-        '12.4'
-        end
-        " >> "$REACT_NATIVE_CODEGEN_UTILS_MODIFIED_PATH"
-        # Modify the get_react_codegen_spec method to return the result using print and JSON.pretty
-        sed -i '' -e "s/:git => ''/:git => 'https:\/\/github.com\/facebook\/react-native.git', :tag => 'v$RN_VERSION'/" "$REACT_NATIVE_CODEGEN_UTILS_MODIFIED_PATH"
-        sed -i '' -e 's/return spec/print JSON.pretty_generate(spec)/' "$REACT_NATIVE_CODEGEN_UTILS_MODIFIED_PATH"
-        # Run get_react_codegen_spec and generate React-Codegen.podspec.json
-        ruby -r "./scripts/cocoapods/codegen_utils_modified.rb" -e "CodegenUtils.new.get_react_codegen_spec('$PACKAGE_JSON_PATH', hermes_enabled:$HERMES_ENABLED)" > "$DEST/React-Codegen.podspec.json"
```

Comment:
Nitpick: After removing this line, we can also remove the `HERMES_ENABLED` variable definition in line 110.

Reply:
Good catch. Thanks!

Address in 5039d5c9935c8c625fbd030d2281822e28cbd81e